apps/web/src/components/professor-card.tsx

```diff
@@ -13,40 +13,44 @@ const CourseAndTutorials = ({ professor }: ProfessorCardProps) => {
   const { t } = useTranslation();
 
   return (
-    <div className="flex flex-wrap content-center items-center gap-2 self-stretch text-base  text-blue-800">
-      <div className="flex items-center gap-2">
-        <div className="font-semibold">{professor.coursesCount}</div>
-        <div className="">{t('words.courses')}</div>
+    <div className="flex content-center items-center gap-2 lg:gap-x-6 self-stretch text-base  text-white">
+      <div className="flex flex-col items-center gap-2">
+        <div className="font-normal text-2xl lg:text-6xl">
+          {professor.coursesCount}
+        </div>
+        <div className="text-xs lg:text-base">{t('words.courses')}</div>
       </div>
-      <span className="text-3xl">•</span>
-      <div className="flex items-center gap-2">
-        <div className="font-semibold">{professor.tutorialsCount}</div>
-        <div className="">{t('words.tutorials')}</div>
+      <div className="flex flex-col items-center gap-2">
+        <div className="font-normal text-2xl lg:text-6xl">
+          {professor.tutorialsCount}
+        </div>
+        <div className="text-xs lg:text-base">{t('words.tutorials')}</div>
       </div>
     </div>
   );
 };
 
 const TopicTags = ({ professor }: ProfessorCardProps) => {
   return (
-    <div className="mt-4 flex flex-wrap  items-start gap-2.5 self-stretch text-xs text-blue-700">
-      {professor.tags?.map((tag) => {
-        return (
-          <div
-            key={tag}
-            className="shadow-md-dark flex items-center rounded-lg bg-gray-100 px-2 py-1"
-          >
-            {tag}
-          </div>
-        );
-      })}
+    <div className="mt-4 flex flex-wrap mx-auto items-center gap-2.5 self-stretch text-xs text-white">
+      {professor.tags?.map((tag) => (
+        <div
+          key={tag}
+          className="shadow-md-dark flex items-center rounded-lg bg-[#FFFFFF40] px-2 py-1"
```

Comment:
We should make it a span and capitalize the tags

Use desktop-typo1 class to manage the font properties